gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@ nva = { strictly = "1.16.30" }
 testUtils = { strictly = "0.1.24" }
 jacksonVersion = { strictly = "2.13.0" }
 jakartaVersion = { strictly = "2.1.6" }
-awsSdkVersion = { strictly = "1.12.129" }
+awsSdkVersion = { strictly = "1.12.134" }
```

Comment:
Is there a reason to not use sdk2?

Reply:
No. You tell me. We just use what is already in use in nva-fetch-doi

Reply:
We should definetely use sdk2

Reply:
I don't think it is a good idea o use different versions of sdk in different modules. I suggest fetch-doi needs a major upgrade first.

Reply:
How should we handle issues like this? 
This is a scope-creep from out Sentralimport perspektive.
Is that a task on NVA board? Do you feel comfortable to outsource the upgrade to us?

Reply:
use skd1